### PR TITLE
fix(rxAutoSave): Update rxAutoSave to use $location.path()

### PR DESCRIPTION
**Pull request by kevincareer427**
JIRA: https://jira.rax.io/browse/FRMW-819
### LGTMs
- [x] Dev LGTM

### CODE REVIEW

File: utils/rx-page-objects/test/rxLocalStorage.midway.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ describe('utilities:rxLocalStorage', function () {
         expect(rxLocalStorage.getItem('anInt')).to.eventually.equal(anInt);
     });
 
-    it('should set and return a float', function () {
+    it.skip('should set and return a float', function () {
```
Comment (Droogans) [inline utils/rx-page-objects/test/rxLocalStorage.midway.js]: If you're going to skip this, you need to track it via a defect in Jira. Once you create it, you can link back to it for future reference by updating the name of this test to:

```js
it.skip('should set and return a float #FRMW-0000', function () {
    // ....
});
```

I can then see stories that are skipped due to open defects quickly and easily in the future!

Comment (kevincareer427) [inline utils/rx-page-objects/test/rxLocalStorage.midway.js]: @Droogans sorry missed that, it's added.

**Comment by Droogans** [comment]
LGTM need a Dev signoff though.

**Comment by CITguy** [comment]
@kevincareer427: please submit PRs against `1.x-LTS` and `2.x-LTS` with these same changes.

**Pull request merged**